build.sbt

```diff
@@ -1,257 +1,77 @@
-import java.text.SimpleDateFormat
-import java.util.Date
-
-import play.sbt.PlayImport
 import sbt.Keys._
 import sbt._
-import sbt.complete.DefaultParsers._
-
-import scala.util.{Success, Try}
+import UnidocKeys._
 
 name := "commercetools-sunrise"
 
 organization in ThisBuild := "com.commercetools.sunrise"
 
-lazy val sunriseThemeVersion = "0.61.1"
+scalaVersion in ThisBuild := "2.11.8"
 
-lazy val jvmSdkVersion = "1.5.0"
+javacOptions in ThisBuild ++= Seq("-source", "1.8", "-target", "1.8")
 
-lazy val jacksonVersion = "2.7.5"
+resolvers in ThisBuild ++= Seq (
+  Resolver.sonatypeRepo("releases"),
+  Resolver.sonatypeRepo("snapshots"),
+  Resolver.mavenLocal
+)
 
+Release.publishSettings
 
-/**
- * SUB-PROJECT DEFINITIONS
- */
+Heroku.deploySettings
 
-lazy val commonWithTests: ClasspathDep[ProjectReference] = common % "compile;test->test;it->it;pt->pt"
+Version.generateVersionInfo
+
+val childProjects: List[sbt.ProjectReference] =
+  List(common, `product-catalog`, `shopping-cart`, `my-account`, `move-to-sdk`, `sbt-tasks`)
 
 lazy val `commercetools-sunrise` = (project in file("."))
-  .enablePlugins(PlayJava, DockerPlugin).configs(IntegrationTest, PlayTest)
-  .settings(commonSettings ++ commonTestSettings ++ sunriseThemeSettings ++ dockerSettings: _*)
-  .dependsOn(commonWithTests, `product-catalog`, `shopping-cart`, `my-account`)
-  .aggregate(common, `product-catalog`, `shopping-cart`, `my-account`, `sbt-tasks`, `move-to-sdk`)
+  .enablePlugins(PlayJava)
+  .settings(javadocSettings ++ Release.disablePublish: _*)
```

Comment:
no need to change it , but splitting it into different settings calls is more consistent

```scala
.settings(javadocSettings: _*)
.settings(Release.disablePublish: _*)
```